### PR TITLE
Feat:  multipart support (Dart+Flutter)

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: templates/dart/lib/src/payload.dart.twig [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  }
+
+  List<int> toBinary() {
+    if (bytes == null) {
+      throw ArgumentError.notNull('bytes');
+    }
+    return bytes!;
+  }
+
+  Map<String, dynamic> toJson() {
```
Comment (abnegate) [inline templates/dart/lib/src/payload.dart.twig:64]: This should be toMap

Comment (Meldiron) [reply templates/dart/lib/src/payload.dart.twig:64]: To have consistency across all languages, I think this should stay as toJson.

**Comment by Meldiron** [comment]
I would recommend to close this in favour of https://github.com/appwrite/sdk-generator/pull/980

**Comment by lohanidamodar** [comment]
Fixed in 
https://github.com/appwrite/sdk-generator/pull/980

**Pull request closed without merging**